docs/integrations/prefect-aws/ecs_guide.mdx

```diff
@@ -93,38 +93,77 @@ If you prefer infrastructure as code check out the [Prefect ECS worker Terraform
 
 ## Prerequisites
 
-- An AWS account with permissions to create ECS services and IAM roles.
+You will need the following to successfully complete this guide:
+
+- A [Prefect Cloud](https://app.prefect.cloud) account or [Prefect self-managed instance](/v3/concepts/server).
```

Comment:
Might be worth noting that you'll need at least a Starter account for Prefect Cloud.